eclair-core/src/main/scala/fr/acinq/eclair/wire/protocol/OfferCodecs.scala

```diff
@@ -130,16 +129,15 @@ object OfferCodecs {
   })
 
   private val paymentInfo: Codec[PaymentInfo] = (("fee_base_msat" | millisatoshi32) ::
-    ("fee_proportional_millionths" | tu32) ::
-    ("cltv_expiry_delta" | cltvExpiryDelta)).as[PaymentInfo]
+    ("fee_proportional_millionths" | uint32) ::
+    ("cltv_expiry_delta" | cltvExpiryDelta) ::
+    ("htlc_minimum_msat" | millisatoshi) ::
+    ("htlc_maximum_msat" | millisatoshi) ::
+    ("features" | variableSizeBytesLong(uint32, bytes).xmap[Features[Feature]](Features(_), _.toByteVector))).as[PaymentInfo]
```

Comment:
Why is this using a `uint32` for the length whereas everywhere else we use `uint16` for features?

Reply:
It was a mistake. Fixed